.github/workflows/generate-api-diffs.yml

```diff
@@ -0,0 +1,30 @@
+name: GenerateAPIDiffs
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 16 * * *' # 8am PST (16:00 UTC)
```

Comment:
we can trigger it based on the changes to PublicAPI.Unshipped.txt (though I want to delete those files 😄).

Reply:
Yeah, once we have this I don't see much point of keeping the publicApiAnalyzers, but curious te see others feedback. @eerhardt thoughts?

Reply:
Removed in [Remove PublicApiAnalyzer (dotnet/aspire#7389)](https://github.com/dotnet/aspire/pull/7389).